WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -3,21 +3,24 @@ package org.wordpress.android.viewmodel.main
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import org.wordpress.android.BuildConfig
 import org.wordpress.android.R
 import org.wordpress.android.ui.main.MainActionListItem
 import org.wordpress.android.ui.main.MainActionListItem.ActionType
 import org.wordpress.android.ui.main.MainActionListItem.ActionType.CREATE_NEW_PAGE
 import org.wordpress.android.ui.main.MainActionListItem.ActionType.CREATE_NEW_POST
 import org.wordpress.android.ui.main.MainActionListItem.CreateAction
+import org.wordpress.android.ui.main.MainFabUiState
+import org.wordpress.android.ui.prefs.AppPrefsWrapper
 import org.wordpress.android.viewmodel.Event
 import org.wordpress.android.viewmodel.SingleLiveEvent
 import javax.inject.Inject
 
-class WPMainActivityViewModel @Inject constructor() : ViewModel() {
+class WPMainActivityViewModel @Inject constructor(private val appPrefsWrapper: AppPrefsWrapper) : ViewModel() {
     private var isStarted = false
 
-    private val _showMainActionFab = MutableLiveData<Boolean>()
-    val showMainActionFab: LiveData<Boolean> = _showMainActionFab
+    private val _showMainActionFab = MutableLiveData<MainFabUiState>()
+    val showMainActionFab: LiveData<MainFabUiState> = _showMainActionFab
```

Comment:
I'd also rename this field 👍

Reply:
Correct 👍...see here 95381cb8653731e40440a61be244ed2eeafd7664